Makefile.include

```diff
@@ -495,9 +495,10 @@ pkg-build-%: $(BUILDDEPS)
 	$(QQ)"$(MAKE)" -C $(RIOTPKG)/$*
 
 clean:
-	-@for i in $(USEPKG) ; do "$(MAKE)" -C $(RIOTPKG)/$$i clean ; done
-	-@rm -rf $(BINDIR)
-	-@rm -rf $(SCANBUILD_OUTPUTDIR)
+	sleep 2
```

Comment:
waitaminute, two seconds for every clean invocation?

Reply:
ok, this is part of the debug commit. nevermind.

Reply:
It is a throwaway commit to force the error to show up.